unity/unity_utils.c

```diff
@@ -198,3 +198,12 @@ unity_mono_method_is_generic (MonoMethod* method)
 {
 	return method->is_generic;
 }
+
+MonoMethod*
+unity_mono_reflection_method_get_method(MonoReflectionMethod* mrf)
+{
+	if(!mrf)
```

Comment:
It's probably fine, but I don't think the embedding API generally handles NULL parameters.